CMakeLists.txt

```diff
@@ -83,6 +83,11 @@ find_package(Snappy REQUIRED)
 find_package(RocksDB REQUIRED)
 find_package(Crypto REQUIRED)
 find_package(Dl REQUIRED)
+
+if(UBUNTU)
+  find_package(AptPkg REQUIRED)
```

Comment:
Nice! But we shouldn't need another Find package include/module for something that's not-so-complicated. I think this is an osquery limitation, not a critique of you including the FindAptPkg, haha. And I call this not-so-complicated because the result CMakeCache variables: `APTPKG_LIBRARIES`, `APTPKG_INCLUDE_DIR` aren't used.

A CMake module for path discovery would be cool for weird inconsistencies in header location or a preferred/local/vs. default location for the static/dynamic dependent library. 

I think we can solve this problem with a better search using CMake's internal `find_library` function. Let me work on this real fast.